src/sardana/macroserver/macros/examples/parameters.py

```diff
@@ -189,7 +309,11 @@ def run(self, *args, **kwargs):
 
 class pt13(Macro):
     """Macro with list of motors groups, where each motor group is a list of 
-    motors. Repeat parameters may be defined as nested."""
+    motors. Repeat parameters may be defined as nested.
+    Usages, ex.:
+    pt13 [[[[mot1] [mot2]]][[[mot3] [mot4]]]]
```

Comment:
After implementing changes agreed in the last #405 comments this should not be accepted. Please remove if you agree.